qemu/tests/cfg/qemu_disk_img_create.cfg

```diff
@@ -0,0 +1,51 @@
+- qemu_disk_img_create:
+    no qed raw vmdk
+    type = create_external_snapshot_via_driver
+    kill_vm = yes
+    start_vm = no
+    md5sum_bin = "md5sum"
+    force_create_image = no
+    backup_image_before_testing = yes
+    restore_image_before_testing = yes
+    tmp_dir = /var/tmp
+    image_chain = "image1 sn1"
+    image_name_sn1 = "images/sn1"
+    image_format_sn1 = "qcow2"
+    tmp_file_name = ${tmp_dir}/test.img
+    guest_temp_file_sn1 = "${tmp_dir}/sn1"
+    guest_temp_file_image1 = "${tmp_dir}/sn1"
```

Comment:
For windows guest, we need to reassign the value of md5sum_bin, guest_temp_file_sn1, guest_temp_file_image1. I think you can refer the definition below qemu_disk_img in the file avocado-vt/shared/cfg/guest-os/Windows.cfg.